src/etc/inc/system.inc

```diff
@@ -1721,8 +1788,8 @@ function system_ntp_configure() {
 
 	/* Add PPS configuration */
 	if (is_array($config['ntpd']['pps']) && !empty($config['ntpd']['pps']['port']) &&
-	    file_exists('/dev/'.$config['ntpd']['pps']['port']) &&
-	    system_ntp_setup_pps($config['ntpd']['pps']['port'])) {
+		file_exists('/dev/'.$config['ntpd']['pps']['port']) &&
+		system_ntp_setup_pps($config['ntpd']['pps']['port'])) {
```

Comment:
You broke style here. Multiline statements should use 4 spaces indent as it was before. Please revert